src/models/offer.js

```
@@ -54,10 +78,14 @@ const offerSchema = new Schema(
       type: [
         {
           question: {
-            type: String
+            type: String,
+            required: [true, FIELD_CANNOT_BE_EMPTY('FAQ-question')],
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)